src/ddmd/backend/cod3.c

```diff
@@ -799,7 +796,6 @@ void outblkexitcode(block *bl, code* c, int& anyspill, const char* sflsave, symb
             }
             logexp(cdb,e,jcond,FLblock,(code *) bs1);
             nextb = bs2;
-            bl->Bcode = NULL;
```

Comment:
Won't this change be visible outside this function?  Why wouldn't we care about Bcode's value?